docs/querying/filters.md

```diff
@@ -352,18 +310,154 @@ Likewise, this filter expresses `age >= 18`
 ```
 
 
+## Range filter
+
+The range filter is a replacement for the bound filter with the ability to compare against any type of column. The range filter has more SQL compliant behavior, and will never match NULL values, even if no lower bound is specified.
+
+Druid's SQL planner will by default use the range filter instead of bound filter whenever `druid.generic.useDefaultValueForNull=false`, or if `sqlUseBoundAndSelectors` is set to false on the [SQL query context](./sql-query-context.md).
```

Comment:
```suggestion
Druid's SQL planner uses the range filter by default instead of bound filter whenever `druid.generic.useDefaultValueForNull=false`, or if `sqlUseBoundAndSelectors` is set to false on the [SQL query context](./sql-query-context.md).
```